src/DynamoUtilities/TestUtilities.cs

```diff
@@ -3,13 +3,30 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace DynamoUtilities
 {
     internal class TestUtilities
     {
+        internal static bool RunningFromNUnit = false;
+        static TestUtilities()
+        {
+            foreach (Assembly assem in AppDomain.CurrentDomain.GetAssemblies())
+            {
+                // Can't do something like this as it will load the nUnit assembly
```

Comment:
What about using a metadata load context - you shouldn’t need to resolve assemblies. Maybe its overkill.

Reply:
removed